core/internal/cltest/cltest.go

```diff
@@ -142,6 +142,15 @@ func init() {
 	source = rand.NewSource(seed)
 }
 
+func NewGomegaWithT(t testing.TB) *gomega.WithT {
+	g := gomega.NewWithT(t)
+	g.SetDefaultEventuallyTimeout(DefaultWaitTimeout)
+	g.SetDefaultEventuallyPollingInterval(DBPollingInterval)
+	g.SetDefaultConsistentlyDuration(time.Second)
+	g.SetDefaultConsistentlyPollingInterval(100 * time.Millisecond)
```

Comment:
Overriding defaults of 1s/10ms and 100ms/10ms.

Reply:
Any reason not to loosen these? It should improve consistency of time sensitive tests with minimal runtime impact.